tests/integration/package.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
This files does not match the `integration` setup.

1. The name property is e2e.
2. There is no `index.js` but it's indicated as the main file.
3. Scripts need revision. For example, there is no script `test:api:all` but `test:watch` depends on it.
4. Some dependencies are unrelated. Example, xlsx, yargs.

Reply:
The name property is e2e.